### PR TITLE
Correct the certificate used to sign EE binaries

### PR DESCRIPTION
**Pull request by jaredpar**
Several of our EE binaries needed to be signed with different certificates than the standard Microsoft one.  Functionality was added to specify the certificate for the binary in question and all remaining references to the AuthenticodeCertificateName property were removed.

### CODE REVIEW

**Comment by jaredpar** [comment]
CC @MattGertz  for ask mode approval

**Comment by jaredpar** [comment]
CC @dotnet/roslyn-infrastructure

File: src/Tools/SignRoslyn/BinaryData.json [hunk 1]
```diff
@@ -1,83 +1,109 @@
...
+                "Roslyn.VisualStudio.Setup.Next.vsix",
+                "Roslyn.VisualStudio.Setup.vsix",
+                "RoslynDeployment.vsix",
+                "SDK\\Roslyn Templates\\Release\\Roslyn SDK.vsix"
+            ]
+        }
     ],
 
-    // Binaries which are included in VSIX above which are not to be signed. 
+    // Binaries which are included in VSIX above which are not to be signed.
```
Comment (tannergooding) [inline src/Tools/SignRoslyn/BinaryData.json]: Same as with the other PR. JSON doesn't support comments.

Comment (jasonmalinowski) [inline src/Tools/SignRoslyn/BinaryData.json]: If you're reading with Newtonsoft.Json, it'll work fine.

**Pull request merged**